pkgs/top-level/aliases.nix

```
@@ -911,6 +912,7 @@ mapAliases ({
   llvm_11 = throw "llvm_11 has been removed from nixpkgs"; # Added 2024-01-24
 
   lobster-two = google-fonts; # Added 2021-07-22
+  lsh = throw "lsh has been removed as it is unmaintained and hasn't seen an upstream release in over a decade"; # Added 2024-08-14
```

Nit:

```suggestion
  lsh = throw "lsh has been removed as it had no maintainer in Nixpkgs and hasn't seen an upstream release in over a decade"; # Added 2024-08-14
```

Just to be slightly more precise about the situation, since there is unreleased upstream activity in Git.